spec/features/flows/sp_authentication_flows_spec.rb

```diff
@@ -58,7 +58,11 @@
 
                   context 'with a valid phone number' do
                     before do
-                      fill_in 'user_phone_form_phone', with: Faker::PhoneNumber.cell_phone
+                      phone = Faker::PhoneNumber.cell_phone
+                      until PhonyRails.plausible_number? phone, country_code: :us
```

Comment:
Can we break out a helper method to generate a number like this? With `git grep Faker::PhoneNumber.cell_phone` I find two uses in this file

Reply:
Moved this into a helper at the end of the spec. PTAL!